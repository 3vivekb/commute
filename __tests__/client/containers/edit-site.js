/* global describe, expect, it */

import { mount } from 'enzyme'
import { mountToJson } from 'enzyme-to-json'
import React from 'react'
import { Provider } from 'react-redux'

import { makeMockStore, mockStoreData } from '../../test-utils/mock-store.js'

import EditSite from '../../../client/containers/edit-site'

const mockStore = makeMockStore(mockStoreData)

describe('Container > EditSite', () => {
  it('renders correctly', () => {
    // mount component
    const tree = mount(
      <Provider store={mockStore}>
        <EditSite />
      </Provider>
    )
    expect(mountToJson(tree.find(EditSite))).toMatchSnapshot()
  })
})
