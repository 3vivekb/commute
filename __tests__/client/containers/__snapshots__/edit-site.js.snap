exports[`Container > EditSite renders correctly 1`] = `
<Connect(EditSite)>
  <EditSite
    dispatch={[Function]}>
    <h1>
      Edit Site
    </h1>
  </EditSite>
</Connect(EditSite)>
`;
